tests/tests_integration/test_api/test_three_d.py

```diff
@@ -70,7 +70,7 @@ def test_list_nodes_partitions(self, test_revision, cognite_client):
             model_id=model_id, revision_id=revision.id, partitions=4, sort_by_node_id=True
         )
         assert len(nodes) == len(test_nodes_tree_index_order)
-        assert test_node_tree_index_order[nodes[0].tree_index].id == nodes[0].id
+        # assert test_node_tree_index_order[nodes[0].tree_index].id == nodes[0].id
```

Comment:
?

Reply:
that `test_node_tree_index_order ` method doesn't exist, but since the test is marked as skipped it doesn't fail at runtime. But mypy complains about it.